CHANGELOG.md

```diff
@@ -75,6 +75,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - [#7716](https://github.com/apache/trafficcontrol/pull/7716) *Apache Traffic Server* Use GCC 11 for building.
 
 ### Fixed
+- [#7734](https://github.com/apache/trafficcontrol/pull/7734) *Traffic Ops* *Traffic Ops* Fixed `Profiles` V5 apis to respond with `RFC3339` date/time Format.
```

Comment:
Extra `*Traffic Ops*`

also, the route is `/profiles` with a lowercase "p", "API" should be all caps, and "Format" need not start with a capital letter.